CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Consider `TAGS` and `GTAGS` root markers.
 * Add relation between the `.h`, `.cxx`, `.ixx` and `.hxx` files in `projectile-other-file-alist`.
 * Add support to specify project name either via `.dir-locals.el` or by providing a customized `projectile-project-name-function'.
+* Use external find-tag implementations.
```

Comment:
To some extent we've done this in the past well, so here you should probably say "always use"...

This should be in changes, not new features. Guess you can add two entries - one about the etags-select here and one about dropping projectile's tag parsing in changes.